jf_agent/main.py

```diff
@@ -612,6 +605,21 @@ def download_data(config, creds, endpoint_jira_info, endpoint_git_instances_info
                 )
                 logger.debug(traceback.format_exc())
                 logger.info(f'Rolling back and using load_and_dump_jira to ingest data')
+                jf_ingest_files = f'{config.outdir}/jira'
+                try:
+                    if os.path.isdir(jf_ingest_files):
+                        logger.debug(
+                            f'JF Ingest files were detected at {jf_ingest_files}, attempting to remove them now'
+                        )
+                        shutil.rmtree(jf_ingest_files)
+                        logger.debug(f'Successfully removed left over JF Ingest files')
+                    else:
+                        logger.debug(f'No JF Ingest files were detected at {jf_ingest_files}')
+                except Exception as e:
+                    # Extreme precaution, wrap the above file operations in a try catch block
+                    logger.debug(
+                        f'Error encountered when attempting to remove JF Ingest files: {e}'
+                    )
```

Comment:
This is the only 'change'. Seems like my formatter went crazy on random stuff in this file 🤔